public/styles/css/main.css

```diff
@@ -2,76 +2,126 @@
 *            Helpers           *
 ********************************/
 .italic {
-  font-style: italic;
-  color: #202020; }
+  font-style: italic; }
 
 .bold {
   font-weight: 700; }
 
 .sub-heading {
   font-size: 2.2rem; }
 
+.highlight {
+  color: #FF6F61;
+  font-weight: 700; }
+
+.size20 {
+  font-size: 20px; }
+
+.fas {
+  color: #FF6F61;
+  font-size: 2em;
+  position: relative;
+  left: 50%;
+  transform: translateX(-50%);
+  margin-top: 15%;
+  margin-bottom: 10%; }
+
 /*******   END HELPERS   *******/
 body {
-  color: #3B3D3E;
-  background-color: #FFFEFD; }
+  color: #393D3F;
+  background-color: #FFF8F8;
+  margin: 10px 10px 10px 10px; }
 
 .container {
-  margin: 5px 10px; }
+  margin: 10px 10px 10px 10px; }
+
+/*******************************
+*         Landing Page         *
+********************************/
+header {
+  position: relative;
+  top: 29px; }
 
 h1 {
-  font-family: "Roboto", sans-serif;
-  color: #202020;
-  font-size: 3.5rem;
-  margin-top: 100px;
-  margin-left: 20px;
+  position: relative;
+  font-family: "Playfair Display", serif;
+  color: #393D3F;
+  font-size: 2.8rem;
+  letter-spacing: 0.1em;
+  margin-top: 40%;
+  margin-left: 4%;
+  line-height: 38px;
   z-index: 100;
-  position: relative; }
-
-.explore-h1 {
-  font-family: "Roboto", sans-serif;
-  color: #202020;
-  font-size: 2.2rem;
-  margin: 0;
-  margin-top: 20px;
-  margin-left: 20px; }
+  top: -1px; }
 
 h2 {
-  font-family: "Roboto", sans-serif;
   margin: 20px 15px; }
 
+#sub-title-container {
+  position: relative;
+  top: -23px;
+  left: 102px; }
+
+#border {
+  border: 2px solid #393D3F;
+  border-radius: 10px;
+  height: 199px;
+  width: 210px;
+  position: relative;
+  left: -24px; }
+
+#border-breaker {
+  height: 180px;
+  width: 214px;
+  background-color: #FFF8F8;
+  position: relative;
+  top: -265px;
+  left: -31px; }
+
 .sub-title {
-  color: #3B3D3E;
-  width: 280px;
-  height: 250px;
+  font-family: "Raleway", sans-serif;
+  font-size: 1.2rem;
+  font-weight: 700;
+  text-transform: uppercase;
+  letter-spacing: 0.2em;
```

Comment:
uppercase can be made into a helper class

Reply:
okie dokie!